{{ cookiecutter.__repo_name }}/{{ cookiecutter.__app_name }}/config.py

```diff
@@ -10,20 +10,34 @@
 import os
 from importlib import metadata
 
-# Get AI model metadata
+import yaml
+
+
+# Get AI model metadata from pyproject.toml
 API_NAME = "{{ cookiecutter.__app_name }}"
-API_METADATA = metadata.metadata(API_NAME)  # .json
+PROJECT_METADATA = metadata.metadata(API_NAME)  # .json
 
 # Fix metadata for emails from pyproject parsing
-_EMAILS = API_METADATA["Author-email"].split(", ")
+_EMAILS = PROJECT_METADATA["Author-email"].split(", ")
 _EMAILS = map(lambda s: s[:-1].split(" <"), _EMAILS)
-API_METADATA["Author-emails"] = dict(_EMAILS)
+PROJECT_METADATA["Author-emails"] = dict(_EMAILS)
 
 # Fix metadata for authors from pyproject parsing
-_AUTHORS = API_METADATA.get("Author", "").replace(", ", ",").split(",")
+_AUTHORS = PROJECT_METADATA.get("Author", "").replace(", ", ",").split(",")
 _AUTHORS = [] if _AUTHORS == [""] else _AUTHORS
-_AUTHORS += API_METADATA["Author-emails"].keys()
-API_METADATA["Authors"] = sorted(_AUTHORS)
+_AUTHORS += PROJECT_METADATA["Author-emails"].keys()
+PROJECT_METADATA["Authors"] = sorted(_AUTHORS)
+
+# Get ai4-metadata.yaml metadata
+AI4_METADATA_DIR = os.getenv(
```

Comment:
I don't understand. The metadata yml file is always going to be in the root dir of the module. You should be able to do this right?
```python
import pathlib
AI4_METADATA_DIR = pathlib.Path(__file__).parent.resolve().parent
```
No need to define env variables.

Reply:
When installed without -e it will be in site-packages folder, so that path will not work. This is for example when using tox

Reply:
Indeed, sorry I forgot we already spoke about this.